applications/StructuralMechanicsApplication/python_scripts/convergence_criteria_factory.py

```diff
@@ -28,8 +28,13 @@ def __init__(self, convergence_criterion_parameters):
             if(convergence_criterion_parameters["rotation_dofs"].GetBool()):
                 rotation_dofs = True
 
+        volumetric_strain_dofs = False
+        if convergence_criterion_parameters.Has("volumetric_strain_dofs"):
+            if convergence_criterion_parameters["volumetric_strain_dofs"].GetBool():
+                volumetric_strain_dofs = True
+
         # Convergence criteria if there are rotation DOFs in the problem
-        if(rotation_dofs is True):
+        if(rotation_dofs or volumetric_strain_dofs):
```

Comment:
don't you in this case want to split the strain dofs?

Reply:
Now I see. Thanks!

Reply:
Done.

Reply:
The comment should be updated to "# Convergence criteria if there are additional DOFs in the problem", or something like that

Reply:
> The comment should be updated to "# Convergence criteria if there are additional DOFs in the problem", or something like that

Sorry, I don't get your point.

Reply:
![image](https://user-images.githubusercontent.com/19991680/69334660-84ee7a00-0c5b-11ea-9800-00687c9bc7e7.png)
Change the first line

Reply:
he changed it already, now is correct IMO